tests/integration/components/frost-object-browser-test.js

```diff
@@ -59,33 +55,17 @@ describe(test.label, function () {
       expect(this.$('.frost-object-browser-facets-header')).to.have.text('Refine by')
     })
 
-    it('with filterVisible=true, should display facets-expanded-view', function () {
+    it('without isFilterHiddenOnLoad, should display facets-expanded-view', function () {
```

Comment:
Test description should be with `should` (this will be enforced via linting in the near future)